test/references/heatmap/colormap_31x31_snow.txt

```diff
@@ -0,0 +1,15 @@
+      [90m┌───────────────────────────────┐[39m          
```

Comment:
Where are the codes that generate these three references?

Reply:
https://github.com/JuliaPlots/UnicodePlots.jl/blob/5beb0b6f369bb060516970bad703aab3d5959453/test/tst_heatmap.jl#L39